docs/notebooks/index.rst

```diff
@@ -0,0 +1,32 @@
+
+.. _notebook_gallery:
+
+Notebook gallery
+================
+
+
+
+Deep learning
+-------------
+
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="Resnet example with Flax and JAXopt.">
+
+.. only:: html
+
+ .. figure:: /notebooks/thumbnails/resnet_flax.png
```

Comment:
```suggestion
 .. figure:: /notebooks/deep_learning/resnet_flax_thumbnail.png
```
This way, the thumbnails follow the same folder hierarchy.

Reply:
that clashes with the jupytext --sync command. But we could have them in `/notebooks/deep_learning/thumbnails/file.png`, what do you think?

Reply:
Sounds good.

Reply:
done